app/controllers/audits/guidances_controller.rb

```diff
@@ -1,7 +1,21 @@
 module Audits
   class GuidancesController < BaseController
     def show
-      @content = Support.get
+      @content = Govspeak::Document.new(Support.get).to_html
+      @headings = h2_from_html(@content)
+    end
+
+  private
+
+    def h2_from_html(html)
```

Comment:
I would prefer not having HMLT related code in a controller. 
If the headings are important, we could extract them within:

```ruby
@content = Support.get
heading = @content.headings
```
Although the best option to me would be to use a custom template renderer. 
Like: `guidance.html.markdown`

Reply:
@pmanrubia I'm not sure I understand the "custom template renderer" part of your comment, but I've moved the HTML logic into the (renamed) `Guidance` class.